app/controllers/posts_controller.rb

```diff
@@ -1,31 +1,61 @@
 class PostsController < ApplicationController
+  load_and_authorize_resource
+
   def index
     @posts = Post.all
     @user = User.includes(posts: [:comments]).find(params[:user_id].to_i)
   end
 
+  def current_ability
+    @current_ability ||= Ability.new(current_admin)
+  end
```

Comment:
This method is also unnecessary since you do not really have to do anything on the post controller except include `load_and_authorize_resource`; which you have, so that should be good enough.

Reply:
When I remove the method from my posts_contoller.rb , it presents me with the error below:


![Screenshot (111)](https://user-images.githubusercontent.com/63410324/201481318-2225ab2e-99f4-4c1a-96cf-3e62adb3ab33.png)


Adding that method communicates with my current_admin since the devise name is 'admin' and not 'user'.